frontend/public/components/dashboard/project-dashboard/getting-started/GettingStartedSection.tsx

```diff
@@ -34,7 +31,7 @@ export const GettingStartedSection: React.FC = () => {
   }
 
   return (
-    <div className="odc-add-page-getting-started-section">
+    <div className="co-add-page-getting-started-section">
```

Comment:
the class [`co-dashboard-getting-started-section`](https://github.com/openshift/console/blob/76e921e3dff48bd0707b05097b308c77886179c1/frontend/public/components/dashboard/dashboards-page/cluster-dashboard/getting-started/getting-started-section.scss#L3) already exists and has the same styling, do we want to deduplicate this now that it belongs to the admin view?